shared/connection.go

```diff
@@ -33,6 +33,10 @@ const (
 
 func RedactMongoUri(uri string) string {
 	if strings.HasPrefix(uri, "mongodb://") && strings.Contains(uri, "@") {
+		if strings.Contains(uri, "ssl=true") {
```

Comment:
It's better to use `net/url.Parse()`, `net/url.URL.Query()` and `strconv.ParseBool()`.